scripts/dotnet-cli-build/SharedFrameworkTargets.cs

```diff
@@ -111,11 +116,15 @@ private static string GetVersionFromProjectJson(string pathToProjectJson)
                 }
             }
 
-            return null;
+            throw new InvalidOperationException("Unable to match the version name from " + pathToProjectJson);
         }
 
-        private static void CrossGenAllManagedAssemblies(string pathToAssemblies)
+        [Target]
+        [Environment("CROSSGEN_SHAREDFRAMEWORK")]
```

Comment:
Make sure that is run only when set to "true" or 1. https://github.com/dotnet/cli/blob/rel/1.0.0/scripts/dotnet-cli-build/PublishTargets.cs#L40